README.md

```diff
@@ -22,7 +22,12 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+var author = AuthorModel.find({name: "Bob"}, function (err, docs){
+  if(err)
+console.log("Action cannot be completed")});
+console.log(docs);
+});
+
```

Comment:
The code in the question is wrong because Mongoose is asynchronous, meaning that anything you want to happen as a result of a Mongoose query must be wrapped in a callback or promise.